openwisp_utils/admin_theme/static/admin/js/ow-auto-filter.js

```diff
@@ -2,15 +2,17 @@
 django.jQuery(document).ready(function () {
   // unbinding default event handlers of admin_auto_filters
   django.jQuery("#changelist-filter select, #grp-filters select").off("change");
-
+  django.jQuery(".auto-filter .select2-selection__placeholder").text("All");
```

Comment:
`gettext("All")`

Probably this has to be called also after the filter selection is cleared (make a reusable function in that case please).